ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -271,9 +287,22 @@ lazy_static::lazy_static! {
         "icu::locid::subtags",
         "icu::locid::LanguageIdentifier",
 
-        // We currently do support this, but diplomat panics on the doc specifier
-        // https://github.com/rust-diplomat/diplomat/pull/244
-        "icu::locid::Locale::UND",
+        // experimental
+        "icu::normalizer::ComposingNormalizer::try_new_uts46_without_ignored_and_disallowed_unstable",
+
+        // can't be exposed till Diplomat has Write16
+        "icu::normalizer::ComposingNormalizer::normalize_utf16",
+        "icu::normalizer::ComposingNormalizer::normalize_utf16_to",
+        "icu::normalizer::ComposingNormalizer::is_normalized_utf16",
+        "icu::normalizer::DecomposingNormalizer::normalize_utf16",
+        "icu::normalizer::DecomposingNormalizer::normalize_utf16_to",
+        "icu::normalizer::DecomposingNormalizer::is_normalized_utf16",
+
+        // Can't be exposed till diplomat has input *and* output iterators
```

Comment:
Nit: We already have an established pattern for output iterators; see BreakIterator. Input iterators are the problem here.

Also, observation, `Composition::compose` is a function that doesn't return an iterator, but the only way to get a `Composition` is to have an input iterator. Should we expose `Composition::compose` somehow?